legal-api/src/legal_api/resources/business/business_filings.py

```diff
@@ -88,28 +83,25 @@ def get(identifier, filing_id=None):  # pylint: disable=too-many-return-statemen
             return jsonify(filings=[]), HTTPStatus.NOT_FOUND
 
         if filing_id:
-            rv = db.session.query(Business, Filing). \
-                filter(Business.id == Filing.business_id).\
-                filter(Business.identifier == identifier).\
-                filter(Filing.id == filing_id).\
-                one_or_none()
-            if not rv:
+            rv = CoreFiling.get(identifier, filing_id)
+            if not rv.storage:
                 return jsonify({'message': f'{identifier} no filings found'}), HTTPStatus.NOT_FOUND
 
             if str(request.accept_mimetypes) == 'application/pdf':
                 report_type = request.args.get('type', None)
-                if rv[1].filing_type == 'incorporationApplication':
-                    ListFilingResource._populate_business_info_to_filing(rv[1], business)
-                return legal_api.reports.get_pdf(rv[1], report_type)
-            return jsonify(rv[1].json)
+                if rv.filing_type == 'incorporationApplication':
+                    ListFilingResource._populate_business_info_to_filing(rv.storage, business)
+                return legal_api.reports.get_pdf(rv.storage, report_type)
```

Comment:
Please validate whether the IA application is getting downloaded after this change.

Reply:
Will do

Reply:
What I meant was with this report will still use the filing model  and it does not use the logic of raw or versioned json at this point.   There will be refactoring needed to make that work

Reply:
Another thing to verify is that UI continues to display incomplete and complete filings properly with this change atleasr for IA. You can run the UI locally pointing to these changes to test that